blocks/footer/footer.css

```diff
@@ -1,85 +1,179 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword, no-descending-specificity */
+
 footer {
-  background-color: var(--light-color);
   font-size: var(--body-font-size-xs);
   border-top: 1px solid var(--border-color);
-}
 
-footer .footer > div {
-  margin: auto;
-  max-width: 1200px;
-  padding: 0 24px 24px;
-}
+  ul, li { margin: 0 }
 
-footer .footer p {
-  margin: 0;
-}
+  .footer > div {
+    margin: auto;
+    max-width: 1200px;
+    padding: 0 24px 24px;
 
-@media (width >= 1080px) {
-  footer .footer > div {
-    padding: 0 32px 24px;
+    @media (width >= 1080px) {
+      padding: 0 32px 24px;
+    }
   }
 
-  footer ul:first-of-type {
-    display: flex;
-    justify-content: left;
-    padding: 5rem 0;
-    gap: 5rem;
-  }
+  .footer-utility {
```

Comment:
Thanks for consolidating footer specific css into one specific file. Appreciated.